### PR TITLE
Implement app creation

### PR DESCRIPTION
**Pull request by MP91**
Implements the full app creation process in this template repository.

### CODE REVIEW

**Review by doosuu — CHANGES_REQUESTED** [review]

File: .devcontainer/scripts/onCreateCommand.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo chmod +x /home/vscode/configure-proxies.sh
+sudo /home/vscode/configure-proxies.sh &> /home/vscode/configure-proxies.log
```
[the remark this answers is not in the document]
Comment (MP91) [reply .devcontainer/scripts/onCreateCommand.sh:3]: what lines do you see? for me the log is just empty...

[the remark this answers is not in the document]
Comment (doosuu) [reply .devcontainer/scripts/onCreateCommand.sh:3]: I see a lot of lines starting with "+" after printing the proxy settings, which isnt harmful but doesnt look good.

[the remark this answers is not in the document]
Comment (MP91) [reply .devcontainer/scripts/onCreateCommand.sh:2]: but they should just be in the log or do you still see them in the terminal?

[the remark this answers is not in the document]
Comment (doosuu) [reply .devcontainer/scripts/onCreateCommand.sh:2]: Saw them on the terminal before the file redirect. File redirect is still fine for me to have a "clean" terminal

File: .devcontainer/scripts/postStartCommand.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo "################################"
+echo "#  Creating your Vehicle App   #"
+echo "################################"
+
+velocitas create $VELOCITAS_CREATE_VALUES
```
Comment (doosuu) [inline .devcontainer/scripts/postStartCommand.sh:6]: Call it args/arguments?

```suggestion
velocitas create $VELOCITAS_CREATE_ARGS
```

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -2,5 +2,18 @@
     "name": "Vehicle App Template",
     "image": "ghcr.io/eclipse-velocitas/devcontainer-base-images/base:v0.2",
     "remoteUser": "vscode",
-    "postCreateCommand": "bash .devcontainer/scripts/postCreateCommand.sh"
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "terminal.integrated.defaultProfile.linux": "zsh",
```
[the remark this answers is not in the document]
Comment (MP91) [reply .devcontainer/devcontainer.json:8]: will be installed once the new baseimage is availabel due to the common-utils

File: .devcontainer/scripts/postStartCommand.sh (hunk 2 above)
Comment (MP91) [reply .devcontainer/scripts/postStartCommand.sh]: done

File: .devcontainer/devcontainer.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (MP91) [reply .devcontainer/devcontainer.json:8]: need to wait for the sdk to be released anyhow

File: .devcontainer/devcontainer.json [hunk 4]
```diff
@@ -2,5 +2,18 @@
     "name": "Vehicle App Template",
     "image": "ghcr.io/eclipse-velocitas/devcontainer-base-images/base:v0.2",
     "remoteUser": "vscode",
-    "postCreateCommand": "bash .devcontainer/scripts/postCreateCommand.sh"
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "terminal.integrated.defaultProfile.linux": "bash",
```
Comment (MP91) [inline .devcontainer/devcontainer.json:8]: why not zsh?

File: .devcontainer/scripts/postStartCommand.sh [hunk 5]
```diff
@@ -1,6 +1,36 @@
...
+                  ██ ▓▓ ▓▓  ▓▓ ▒▒ ▓▓▓▓▒▒    ▓▓       ▓▓             ▓▓                             ▓▓                         
+                   ██ ▓▓ ▓▓  ▓▓█▒▒▓▓▓▒▒      ▓▓     ▓▓   ▓▓▓▓▓▓▓    ▓▓   ▓▓▓▓▓▓▓    ▓▓▓▓▓▓▓  ▓▓  ▓▓▓▓▓▓▓   ▓▓▓▓▓▓▓   ▓▓▓▓▓▓▓▓ 
+                    ██ ▓▓ ▓▓  ▓▓█▓▓▓▓         ▓▓   ▓▓▓  ▓▓▓    ▓▓   ▓▓  ▓▓     ▓▓  ▓▓        ▓▓    ▓▓           ▓▓   ▓▓       
+                     ██ ▓▓ ▓▓  ▓▓▓▓▓          ▓▓   ▓▓   ▓▓▓▓▓▓▓▓▓   ▓▓  ▓▓     ▓▓  ▓▓        ▓▓    ▓▓      ▓▓▓▓▓▓▓   ▓▓▓▓▓    
+                      ██ ▓▓ ▓▓██▓▓             ▓▓ ▓▓    ▓▓▓         ▓▓  ▓▓     ▓▓  ▓▓        ▓▓    ▓▓     ▓▓    ▓▓        ▓▓▓ 
+                       ██ ▓▓  ▓▓▓              ▓▓▓▓▓    ▓▓▓         ▓▓  ▓▓▓    ▓▓  ▓▓▓       ▓▓    ▓▓     ▓▓    ▓▓         ▓▓ 
+                        ██ ▓▓ █▓                ▓▓▓       ▓▓▓▓▓▓▓   ▓▓    ▓▓▓▓▓      ▓▓▓▓▓▓  ▓▓      ▓▓▓    ▓▓▓ ▓▓   ▓▓▓▓▓▓   
+                         ██  ██                                                                                               
+                          ████                                                                                                
+"
```
Comment (BjoernAtBosch) [inline .devcontainer/scripts/postStartCommand.sh:18]: Nice :-)

**Review by mml5bg — APPROVED** [review]
LGTM

**Pull request merged**